library/eclipse-mosquitto

```diff
@@ -1,12 +1,9 @@
 Maintainers: Roger Light <roger@atchoo.org> (@ralight)
 GitRepo: https://github.com/eclipse/mosquitto.git
-GitCommit: f4400fa422eacac8417efbc45dd1284526dce8d4
+GitCommit: e96ddfc6e58f7087b9fa57a717d7f56d2c00f9fe
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le
```

Comment:
We disabled your `s390x` builds in https://github.com/docker-library/official-images/pull/17382, but I think the images that are left should work for it now (https://github.com/eclipse/mosquitto/issues/3055). A quick local (emulated) build test works; do you want to add `s390x` back?
```suggestion
Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
```

Reply:
Yes please! I didn't want to complicate the update, but if you're happy it's building then that's ok. I've added the change to the PR.